docs/browse-list.md

```diff
@@ -20,7 +20,23 @@ Expected time: 5.5 - 6 hours.
 
 ## Troubleshooting
 
-When browse lists fail to generate and would like to run the rake task before it is scheduled:
+When **subject lists**, which is used in `Subject (browse)` in Orangelight, fail to generate and would like to run the rake task before it is scheduled: SSH as `deploy` user to the machine used to produce the browse lists (`catalog-indexer1|2|3`):
```

Comment:
It might be helpful to add info about what might lead you to suspect that it didn't run, and how to confirm it.